doc/cephadm/operations.rst

```diff
@@ -97,9 +87,9 @@ To enable logging to files, run the following commands:
 Disabling logging to stderr
```

Comment:
This PR is correct for pacific.  For quincy, though, I think we should revise this to be ``Disabling logging to journald``, since that is effectively where the logs go.  And the settings should include:

```
ceph config set global log_to_file true
ceph config set global log_to_stderr false
ceph config set global log_to_journald false
ceph config set global mon_cluster_log_to_file true
ceph config set global mon_cluster_log_to_stderr false
ceph config set global mon_cluster_log_to_journald false
```

and shoudl mention that on bootstrap, you can pass ``--log-to-file`` and have this set up for you at cluster creation time.

Reply:
I have updated the doc.